### ISSUE TITLE
CSS package

### ISSUE THREAD

**Issue by mimarz** [digdir/designsystem#1045]

**Comment by Barsnes** [comment]
We need to ensure our selectors work in a consisten way.
Sometimes the DOM does not need to follow explicit rules. Tabs, for example, can have a wrapper around the panels.
Our current selectors would break this.

A suggestion from @eirikbacker is to run selectors like
```css
& [role="tabpanel"]:not([role="tabpanel"] [role="tabpanel"])
```

However, places where we want the consumers to keep a specific DOM structure, we should use CSS in a way that would "break" doing other DOM setups.

**Comment by eirikbacker** [comment]
yeah, and we need to test this selector - does it work if using .tabs inside .tabs, because that would actually meet `[role="tabpanel"] [role="tabpanel"]` as well 🤔

**Comment by mrosvik** [comment]
Closing this as we have completed the first version. Further enhancements will be addressed in separate issues.
